src/env.h

```diff
@@ -563,7 +561,6 @@ class Environment {
   inline void FinishHandleCleanup(uv_handle_t* handle);
 
   inline AsyncHooks* async_hooks();
-  inline DomainFlag* domain_flag();
```

Comment:
The `DomainFlag` type is no longer used. Can you remove the class https://github.com/nodejs/node/blob/e3503aca086f3f2f9587004ac594ed83a57f7443/src/env.h#L466 :)

Reply:
Good catch!